releasenotes/notes/add-additional-endpoint-metadata-e76fa630fef107c4.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add additional endpoint metadata to the destination metadata for easier troubleshooting.
```

Comment:
NIT: Since the user facing impact is a change to our logging, I'd either clarify that here ex:
> Improved logs destination warning log with xyz context

But since it's a logging only change, I'd lean to just dropping the change log entry for this.